File: temp/소수찾기(1단계).js
function fn_getPopName(n){
    let cnt = 1;

    function isPrime(num) {

        for (let i = 3; i <= Math.sqrt(num); i += 2) {
            if(num % i ==0){
                return false;
            }
        }
        return true;
    }

    if(n > 2){
        for (let i = 3; i <= n; i += 2) {
            if(isPrime(i)){
                cnt++
            }
        }
    }

    return cnt;
}

File: temp/소수찾기(1단계).java
class Solution {
    public int solution(int n) {
        int answer = 1;
        boolean trigger = true;
        
        for(int i = 3; i <= n; i += 2){
            trigger = true;
            for(int j = 3; j <= Math.sqrt(i); j+=2){
                if(i % j == 0){
                    trigger = false;
                    break;
                }
            }
            if(trigger){
                answer++;
            }
        }
        return answer;
    }
}
